shared/src/akri/configuration.rs

```diff
@@ -36,9 +34,12 @@ pub struct DiscoveryHandlerInfo {
 /// capabilities.  For any specific capability found that is described by this
 /// configuration, an Instance
 /// is created.
-#[derive(Serialize, Deserialize, Clone, Debug)]
+#[derive(CustomResource, Serialize, Deserialize, Clone, Debug, JsonSchema)]
+// group = API_NAMESPACE and version = API_VERSION
+#[kube(group = "akri.sh", version = "v0", kind = "Configuration", namespaced)]
```

Comment:
can these use our constants?  or do constants not work in the attributes?

Reply:
They weren't working which is why I commented it. I'll look into it a little more, though.

Reply:
looks like no: https://github.com/rust-lang/rust/issues/52393